src/main/java/org/granitepowered/granite/impl/GraniteServer.java

```diff
@@ -205,8 +206,9 @@ public int getRunningTimeTicks() {
 
     @Override
     public void broadcastMessage(Message message) {
-        // TODO
-        throw new NotImplementedException("");
+        for (Player p : getOnlinePlayers()){
+            p.sendMessage(ChatTypes.CHAT, message);
+        }
     }
```

Comment:
Not sure but I think there is a broadcast methods built in to minecraft that is used by the console? I may be wrong.